triage-process.md

```diff
@@ -0,0 +1,59 @@
+# High Level Proposal:
+1. Define and publish a process to handle issues in a structured manner.
+    1. Issue Handling Process
+    1. Issue Stages
+    1. Roles
+    1. SLA
+
+## Process Definition
+
+### Issue Handling Process
+1. All incoming issues and PR’s should be as. These are items that a Triager would look at on a daily basis
+1. Every issue triaged should be assessed for its workability. If assessing the workability of the issue requires more information, the Triager would collaborate with the OP until workability is determined.
+    1. Workability Classification:
+        1. New Feature Request
+        1. Enhancement to Existing Feature
+        1. Bug
```

Comment:
Change it to.
1. feature-request
2. enhancement
3. bug

Above are also label names.

Reply:
Done. Also added wont-fix as a label.

Reply:
I think we decided on wontfix (no hyphen) because it is pretty standard on github.

Reply:
Additional classification: docs

Reply:
Done. Also have updated the Issue Stages section. I have kept completed and blocked stages in there since nothign was mentioned explicitly. LMK if you want those removed.

Reply:
We want to keep 'blocked' as a label but 'completed' is unnecessary as a label (it is a stage but is indicated by the issue being closed).

Reply:
Done. blocked is specified in the label column and completed has no label, but it is called out as a stage.